Error during compile with library vertdll

Compile fails with the following error message

```
libvcruntime.lib(memcpy.obj) : error LNK2005: memcpy already defined in vertdll.lib(vertdll.dll)
```

--- a/src/win32cr/system/environment.cr
+++ b/src/win32cr/system/environment.cr
@@ -14,7 +14,9 @@ require "../foundation.cr"
 {% else %}
 @[Link("userenv")]
 @[Link("onecore")]
+{% if flag?(:preview_dll)%}
 @[Link("vertdll")]
+{% end %}
 {% end %}
 lib LibWin32
   ENCLAVE_RUNTIME_POLICY_ALLOW_FULL_DEBUG = 1_u32

--- a/src/win32cr/system/threading.cr
+++ b/src/win32cr/system/threading.cr
@@ -17,7 +17,9 @@ require "../system/systeminformation.cr"
 @[Link(ldflags: "/DELAYLOAD:onecore.dll")]
 @[Link(ldflags: "/DELAYLOAD:user32.dll")]
 {% else %}
+{% if flag?(:preview_dll)%}
 @[Link("vertdll")]
+{% end %}
 @[Link("advapi32")]
 @[Link("onecore")]
 @[Link("user32")]

